safe/gui/tools/minimum_needs/needs_profile.py

```diff
@@ -162,6 +162,15 @@ def sort_by_locale(unsorted_profiles, locale):
         profiles = sort_by_locale(profiles, self.locale)
         return profiles
 
+    def precision_of(self, num_txt):
+        """Get the precision by counting chars to right of '.' .
+
+        :returns: The precision of float parameters.
```

Comment:
:returns: The precision of the passed in textual representation of a number.
   The number of digits after the decimal will be counted and used as returned as the precision.